seatunnel-connectors-v2/connector-iceberg/src/main/java/org/apache/seatunnel/connectors/seatunnel/iceberg/config/SourceConfig.java

```diff
@@ -32,12 +34,35 @@
 public class SourceConfig extends CommonConfig {
     private static final long serialVersionUID = -1965861967575264253L;
 
-    private static final String KEY_START_SNAPSHOT_TIMESTAMP = "start_snapshot_timestamp";
-    private static final String KEY_START_SNAPSHOT_ID = "start_snapshot_id";
-    private static final String KEY_END_SNAPSHOT_ID = "end_snapshot_id";
-    private static final String KEY_USE_SNAPSHOT_ID = "use_snapshot_id";
-    private static final String KEY_USE_SNAPSHOT_TIMESTAMP = "use_snapshot_timestamp";
-    private static final String KEY_STREAM_SCAN_STRATEGY = "stream_scan_strategy";
+    public static final Option<Long> KEY_START_SNAPSHOT_TIMESTAMP = Options.key("start_snapshot_timestamp")
+        .longType()
+        .noDefaultValue()
+        .withDescription(" the iceberg timestamp of starting snapshot ");
+
+    public static final Option<Long> KEY_START_SNAPSHOT_ID = Options.key("start_snapshot_id")
+        .longType()
+        .noDefaultValue()
+        .withDescription(" the iceberg id of starting snapshot ");
+
+    public static final Option<Long> KEY_END_SNAPSHOT_ID = Options.key("end_snapshot_id")
+        .longType()
+        .noDefaultValue()
+        .withDescription(" the iceberg id of ending snapshot ");
+
+    public static final Option<Long> KEY_USE_SNAPSHOT_ID = Options.key("use_snapshot_id")
+        .longType()
+        .noDefaultValue()
+        .withDescription(" the iceberg used snapshot id");
+
+    public static final Option<Long> KEY_USE_SNAPSHOT_TIMESTAMP = Options.key("use_snapshot_timestamp")
+        .longType()
+        .noDefaultValue()
+        .withDescription(" the iceberg used snapshot timestamp");
+
+    public static final Option<Long> KEY_STREAM_SCAN_STRATEGY = Options.key("stream_scan_strategy")
+        .longType()
+        .noDefaultValue()
```

Comment:
<img width="702" alt="image" src="https://user-images.githubusercontent.com/32193458/201452786-e08949c2-4179-4d09-9418-a466681b6845.png">

Reply:
Thank you.I will fix it.